src/components/mods.tsx

```diff
@@ -125,6 +125,7 @@ class ModList extends Component<{selected: string}>
             if (this.props.selected === "Available") {
                 availableList = database!.map((mod) => {
                     if (!info!.has(mod.ModGUID)) {
+                        mod.Author = mod.Repo?.split("/")[0]
```

Comment:
the database info just has an author field now which is this by default and can be overridden when they submit the mod

Reply:
Thought that was the case but wasn't managing to find it-

Just figured out that it's because we forgot to update the ModDatabaseInfo struct in database.rs so it wasn't fetching those new properties